cmd/signals/on_exit.go

```diff
@@ -12,7 +13,7 @@ func GoOnExitSignal(onInterrupt func()) {
 		// Set up sigCh to receive when this process receives an interrupt or
 		// kill signal.
 		sigCh := make(chan os.Signal, 1)
-		signal.Notify(sigCh, os.Interrupt, os.Kill)
+		signal.Notify(sigCh, os.Interrupt, syscall.SIGTERM)
```

Comment:
@bryanchriswhite I was troubleshooting hot reloading issues, and ran into that: https://staticcheck.dev/docs/checks/#SA1016.

I only later realized this code is not used by the sequencer, so I can roll this change back.